src/celeritas/field/RZMapField.hh

```diff
@@ -9,6 +9,8 @@
 
 #include "corecel/Macros.hh"
 #include "corecel/Types.hh"
+#include "corecel/grid/UniformGrid.hh"
+#include "corecel/grid/detail/FindInterp.hh"
```

Comment:
`detail` classes shouldn't be included outside of the direct parent directory. If `FindInterp` rather than using `TwodGridCalculator` is the way to go, could you move that file `corecel/grid`?